From leemgs1@gmail.com  Fri Sep 11 15:04:10 2009
From: GeunSik Lim <leemgs1@gmail.com>
Date: Mon, 07 Sep 2009 21:37:37 +0900
Subject: debugfs: Change debuhgfs directory of trace-events-sample.h
To: Greg KH <greg@kroah.com>
Cc: Steven Rostedt <rostedt@goodmis.org>
Message-ID: <1252327057.2153.97.camel@centos51>


Default directory of debug filesystem for ftrace is /sys/kernel/debug/.

Signed-off-by: GeunSik Lim <geunsik.lim@samsung.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 samples/trace_events/trace-events-sample.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/samples/trace_events/trace-events-sample.h
+++ b/samples/trace_events/trace-events-sample.h
@@ -1,6 +1,6 @@
 /*
  * If TRACE_SYSTEM is defined, that will be the directory created
- * in the ftrace directory under /debugfs/tracing/events/<system>
+ * in the ftrace directory under /sys/kernel/debug/tracing/events/<system>
  *
  * The define_trace.h below will also look for a file name of
  * TRACE_SYSTEM.h where TRACE_SYSTEM is what is defined here.
